app/src/pages/instance/organizationsPage/organizationsPanelView/organizationCard/organizationCard.jsx

```diff
@@ -0,0 +1,126 @@
+/*!
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import classNames from 'classnames/bind';
+import PropTypes from 'prop-types';
+import { useIntl, FormattedRelativeTime } from 'react-intl';
+import Parser from 'html-react-parser';
+import { useSelector } from 'react-redux';
+import { Tooltip } from '@reportportal/ui-kit';
+import { NavLink } from 'components/main/navLink';
+import { ORGANIZATION_PROJECTS_PAGE } from 'controllers/pages/constants';
+import { userRolesSelector } from 'controllers/pages';
+import { MANAGER } from 'common/constants/projectRoles';
+import { ADMINISTRATOR } from 'common/constants/accountRoles';
+import { getRelativeUnits } from 'common/utils/timeDateUtils';
+import UserIcon from './img/user-inline.svg';
+import ProjectsIcon from './img/projects-inline.svg';
+import LastUpdateIcon from './img/last-update-inline.svg';
+import SynchedIcon from './img/synched-organization-inline.svg';
+import OutdatedIcon from './img/outdated-inline.svg';
+import PersonalIcon from './img/personal-organization-inline.svg';
+import { messages } from '../../messages';
+import styles from './organizationCard.scss';
+
+const cx = classNames.bind(styles);
+const EXTERNAL_TYPE = 'EXTERNAL';
```

Comment:
Let's be more descriptive here, e.g. `ORGANIZAION_EXTERNAL_TYPE`.
We can also create a file with constants at the `common/constants` with `organizationTypes` as seems they may be reused later.

Reply:
Done.